workshop/content/60-construct-hub/_index.md

```diff
@@ -0,0 +1,38 @@
++++
+title = "Construct Hub"
+weight = 60
+chapter = true
++++
+
+# The Construct Hub
+
+The [CDK Construct Hub](https://constructs.dev/) is a one-stop destination for finding, reusing and sharing constructs authored by AWS, [AWS Partner Network partners](https://aws.amazon.com/partners/), third parties, and the developer community. The site currently lists constructs for our supported programming languages Typescript, Java, Python, and .NET. CDK constructs are cloud architecture building blocks and patterns that you can use to stand up complete production-ready cloud applications. The constructs listed in the Construct Hub are authored using the [AWS Cloud Development Kit](https://aws.amazon.com/cdk/) (AWS CDK), [CDK for Kubernetes](https://cdk8s.io/) (CDK8s) and [CDK for Terraform](https://github.com/hashicorp/terraform-cdk) (CDKtf).
+
```

Comment:
Should add somewhere in this section [this link](https://docs.aws.amazon.com/cdk/latest/guide/constructs.html)